docker/Dockerfile

```diff
@@ -37,9 +37,9 @@ SHELL ["conda", "run", "-n", "COMPASS", "/bin/bash", "-c"]
 WORKDIR /home/compass_user/OPERA
 
 # installing OPERA s1-reader
-RUN curl -sSL https://github.com/opera-adt/s1-reader/archive/refs/tags/v0.1.5.tar.gz -o s1_reader_src.tar.gz &&\
+RUN curl -sSL https://github.com/opera-adt/s1-reader/archive/main.tar.gz -o s1_reader_src.tar.gz &&\
     tar -xvf s1_reader_src.tar.gz &&\
-    ln -s s1-reader-0.1.5 s1-reader &&\
+    ln -s s1-reader-main s1-reader &&\
```

Comment:
This is better for CI to take the changes in `s1-reader` in to account, but I think we'll need to hard code its version in the upcoming deliveries. @vbrancat @LiangJYu Any opinions?